src/math/math.h

```diff
@@ -0,0 +1,20 @@
+//
+// Created by blinku on 07.02.19.
+//
+
+#pragma once
+
+namespace Bald::Math {
+    template<typename T>
+    constexpr void ConstexprSwap(T& first, T& second) {
```

Comment:
maybe this will by more usefull in utils?

Reply:
Corrected :+1: